package.json

```diff
@@ -18,10 +18,10 @@
   "dependencies": {
     "atom-linter": "^7.0.0",
     "atom-message-panel": "^1.2.4",
-    "atom-package-deps": "^4.0.1",
+    "atom-package-deps": "^4.3.1",
     "which": "^1.2.9"
   },
-  "packages-deps": [
+  "package-deps": [
```

Comment:
I can't tell you how many times I've seen that error, tried to figure out why it was triggering, and missed that extra `s`!